tooling/templatize/pkg/config/config_test.go

```diff
@@ -231,3 +233,78 @@ func TestConvertToInterface(t *testing.T) {
 	assert.IsType(t, expected, map[string]any{})
 	assert.IsType(t, expected["key2"], map[string]any{})
 }
+
+func TestPreprocessContent(t *testing.T) {
+	fileContent, err := os.ReadFile("../../testdata/test.bicepparam")
```

Comment:
Consider `//go:embed` for static test data, much faster.

Reply:
i'll do some readup on go:embed. ty